eth_keys/backends/base.py

```diff
@@ -27,3 +27,11 @@ def ecdsa_recover(self,
     def private_key_to_public_key(self,
                                   private_key: PrivateKey) -> PublicKey:
         raise NotImplementedError()
+
+    def decompress_public_key_bytes(self,
+                                    compressed_public_key_bytes: bytes) -> bytes:
+        raise NotImplementedError()
```

Comment:
Note to self that we should convert these to be `ABC,abstractmethod` based sometime.